library/cpp/include/wavemap/pipeline/pipeline.h

```diff
@@ -26,43 +29,62 @@ class Pipeline {
         thread_pool_(thread_pool ? std::move(thread_pool)
                                  : std::make_shared<ThreadPool>()) {}
 
+  // Copy construction is not supported
+  Pipeline(const Pipeline&) = delete;
+
+  //! Deregister all measurement integrators and map operations
   void clear();
```

Comment:
Why is this called only `clear()` when it handles integrators and the one for operations is called `clearOperations()` wouldn't `clearIntegrators()` not be more consistent?

Reply:
There are three clear methods: `clear()`, `clearIntegrators()`, and `clearOperations()`. The main `clear()` method simply calls the latter two.